tests/integration/test_notes.py

```diff
@@ -50,3 +48,34 @@ def test_it_deletes_one_note(
         mock_delete.assert_called_once_with(note)
         assert response.status_code == 200
         assert response.json == {"message": "Note deleted"}
+
+
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestUpdate:
+    def setup(self):
+        self.endpoint = "/api/tickets"
+        self.new_text = "We don't need no water"
+
+    def test_it_updates_one_note(self, valid_header, create_note):
+        note = create_note()
+
+        response = self.client.patch(
+            f"{self.endpoint}/{note.ticket_id}/notes/{note.id}",
+            json={"text": self.new_text},
+            headers=valid_header,
+        )
+
+        assert response.status_code == 200
+        assert response.json["id"] == note.id
+        assert response.json["text"] == self.new_text
+
+    def test_it_updates_one_note_fails(self, valid_header, create_ticket):
```

Comment:
I don't think this test is necessary.

Reply:
oh I see. Maybe you are trying to test the `find` implementation on NobiruList. Then I think this test is necessary, but it is in the wrong test file.